src/components/ListItem.jsx

```diff
@@ -47,6 +47,12 @@ export function ListItem({
 		setIsChecked(true);
 	};
 
+	const handleDelete = async () => {
+		if (window.confirm('Are you sure you want to delete this item?')) {
+			await deleteItem(listId, itemId);
+		}
+	};
+
```

Comment:
`thought`: perhaps we could consider using a modal for this alert in the future. It's a bit more user-friendly. And we avoid things like this: 
![Screenshot 2023-09-20 at 8 27 21 AM](https://github.com/the-collab-lab/tcl-64-smart-shopping-list/assets/97368143/54cb254c-cf2e-4f50-a532-d8a4d0bb7b94)